src/6_branch.js

```diff
@@ -239,34 +211,12 @@ Branch.prototype._publishEvent = function(event) {
 	}
 };
 
-if (CORDOVA_BUILD || TITANIUM_BUILD) {
-	/** =CORDOVA
-	 * @function Branch.setDebug
-	 * @param {boolean} debug - _required_ - Set the SDK debug flag.
-	 *
-	 * Setting the SDK debug flag will generate a new device ID each time the app is installed
-	 * instead of possibly using the same device id.  This is useful when testing.
-	 *
-	 * This needs to be set before the Branch.init call!!!
-	 *
-	 * ___
-	 *
-	 */
-	Branch.prototype['setDebug'] = function(debug) {
-		this.debug = debug;
-	};
-}
-
 /**
  * @function Branch.init
  * @param {string} branch_key - _required_ - Your Branch [live key](http://dashboard.branch.io/settings), or (deprecated) your app id.
- * @param {{isReferrable:?boolean}=} options - _optional_ - { *isReferrable*: _Is this a referrable session_ }.
```

Comment:
This should stay in.  There are no longer any options, because `isReferrable` was the only attribute, and that in turn was only used by Cordova and Titanium.  However, for safety, the function signature should not change.